uint8 index
bool subindex
bool value
---
string success